codec-http2/src/main/java/io/netty/handler/codec/http2/WeightedFairQueueByteDistributor.java

```diff
@@ -338,5 +381,33 @@ public int priorityQueueIndex() {
         public void priorityQueueIndex(int i) {
             priorityQueueIndex = i;
         }
+
+        @Override
+        public String toString() {
+            StringBuilder sb = InternalThreadLocalMap.get().stringBuilder();
```

Comment:
+1. I was actually thinking about this this morning. Other places where we use this and don't provide any other option may have similar issues.